### ISSUE TITLE
Github can't verify some Simplelogin aliases

### ISSUE THREAD

**Issue by gelatinbomb** [simple-login/app#1631]
Please note that this is only for bug report.

For help on your account, please reach out to us at hi[at]simplelogin.io. Please make sure to check out [our FAQ](https://simplelogin.io/faq/) that contains  frequently asked questions.

For feature request, you can use our [forum](https://github.com/simple-login/app/discussions/categories/feature-request).

For self-hosted question/issue, please ask in [self-hosted forum](https://github.com/simple-login/app/discussions/categories/self-hosting-question)

## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report

**Describe the bug**
I replaced my email here on github with an alias and I noticed "random words" simplelogin aliases pops up an error from Github. It says it cannot get verified. Other aliases like the custom one works.

**Expected behavior**
I expected Github might behave like the rest of websites and accept my alias.

**Environment (If applicable):**
 - OS: Linux
 - Browser: Chrome
 - Version 78

**Additional context**
My theory is the official domain of simplelogin might be blocklisted from Github. Since you are the admin of the site maybe you can talk with Github support for allowing the service. Again aleeas domain works so I will asume something got wrong with the simplelogin domain.

**Thanks for reading and providing this amazing service!**

**Comment by nguyenkims** [comment]
@gelatinbomb thanks for letting us know. This is a known issue and we'll reach out to github to ask for more info.

**Comment by ProPatte** [comment]
Same problem here!

**Comment by zhoull88** [comment]
这是啥

**Comment by tberlin-om** [comment]
Had no luck verifying any of the domains from simplelogin. too bad. Are there any news on this?
I would actually very much like to verify one as my main e-mail-address.

**Comment by gelatinbomb** [comment]
@tberlin-om Did you tried slmails.com? I managed to use that on github with the custom alias.

**Comment by tberlin-om** [comment]
@gelatinbomb : Yes, slmails.com with a custom alias was also removed after verfication.

**Comment by brieucdlf** [comment]
Got this issue as well today with the domain `8shield.net` :( I'll try with other domain if I can pass the verification.

**Comment by stefanthoss** [comment]
`@8alias.com`, `@dralias.com`, `@simplelogin.com`, and `@8shield.net` all give me this error on GitHub. Managed to get a `@slmails.com` address verified.

**Comment by Zhewadachi** [comment]
simplelogin.com doesn't work, but passmail.com works
